lib/relation-mappings/has-one.ts

```diff
@@ -31,4 +33,4 @@ export default function generateHasOneRelationMapping(
   };
```

Comment:
Wouldn't we need to use this new method here?

Reply:
How would it be used here as well?

Reply:
On line 30 above: 

>       from: `${ ObjectionModel.tableName }.${ name }Id`, // i.e. from: 'Comment.postId'

That's defining the name of a foreign key column, right? I would imagine we would need to be invoking your newly added hook there to be consistent.

Reply:
Oh, I see, yeah.